android/src/main/java/com/tailscale/ipn/ui/viewModel/SettingsViewModel.kt

```diff
@@ -3,35 +3,21 @@
 
 package com.tailscale.ipn.ui.viewModel
 
-import androidx.annotation.StringRes
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.res.stringResource
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
-import androidx.lifecycle.viewModelScope
-import com.tailscale.ipn.BuildConfig
 import com.tailscale.ipn.R
-import com.tailscale.ipn.mdm.MDMSettings
 import com.tailscale.ipn.mdm.StringSetting
 import com.tailscale.ipn.ui.notifier.Notifier
 import com.tailscale.ipn.ui.util.set
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
-import kotlinx.coroutines.launch
 
 enum class SettingType {
   NAV,
   SWITCH,
   TEXT
 }
 
-class ComposableStringFormatter(@StringRes val stringRes: Int, vararg val params: Any) {
```

Comment:
This level of indirection seemed unnecessary, as we don't actually use the params anywhere. It's unlikely that this had a performance effect, but one less thing to worry about.